### PR TITLE
Goal kick

### PR DESCRIPTION
**Pull request by dalestee**
If the ball velocity is moving at 10cm per sec inside the ally penalty zone the keeper will go kick it to the closest available ally

### CODE REVIEW

File: crates/crabe_decision/src/strategy/defensive/goal_keeper.rs
Comment (Jossl123) [inline crates/crabe_decision/src/strategy/defensive/goal_keeper.rs:1]: "            let follow_ball = Point2::new(ball_position.x, ball_position.y);"
you can just use ball_position (you probably know it but, if your problem was the the position was 3d, ball.position_2d() gives you the x and y)

**Review by Jossl123 — COMMENTED** [review]
if no ally is available, you should kick to an empty space maybe, (cause we absolutely need to get the ball out of the penalty zone)

File: crates/crabe_decision/src/strategy/defensive/goal_keeper.rs
Comment (dalestee) [reply crates/crabe_decision/src/strategy/defensive/goal_keeper.rs:1]: its true, copy paste issue

**Comment by dalestee** [comment]
> if no ally is available, you should kick to an empty space maybe, (cause we absolutely need to get the ball out of the penalty zone)

done

**Review by Jossl123 — APPROVED** [review]

**Pull request merged**